helpers/encrypt/bcrypt_test.go

```diff
@@ -0,0 +1,29 @@
+package encrypt_test
+
+import (
+	"ca-amartha/helpers/encrypt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var password string
+
+func TestMain(m *testing.M) {
+	password = "secret"
+}
+
+func TestEncrypt(t *testing.T) {
```

Comment:
kayanya ga perlu gini deh.
harusnya pakai ini...

`
t.Run("Hash the password, Valid condition", func(t *testing.T) {
		_, err := encrypt.Hash(password)
		assert.Nil(t, err)
	})
`